builtin/credential/cert/backend.go

```diff
@@ -14,6 +18,9 @@ func Factory(ctx context.Context, conf *logical.BackendConfig) (logical.Backend,
 	if err := b.Setup(ctx, conf); err != nil {
 		return nil, err
 	}
+	if err := b.populateCRLs(ctx, conf.StorageView); err != nil {
```

Comment:
Probably also needs to be called in the invalidate func?

Reply:
I agree. How did this ever work?